docs/README.md

```diff
@@ -57,6 +57,24 @@ Users of the Fuel indexer project include dApp developers looking to write flexi
 - We use Docker to produce reproducible environments for users that may be concerned with installing components with large sets of dependencies (e.g. Postgres).
 - Docker can be downloaded [here](https://docs.docker.com/engine/install/).
 
+### `postgresSQL` and `SQLite`
```

Comment:
@bitzoic 
- I suggest we:
  - Add `PostgreSQL` and `SQlite` sub-headings under `For Users > Dependencies`.
  - Then link those sub-headings here, in `PostgreSQL` and `SQLite` sections
  - Then for now, we can just link to the download page (indexer team can follow up with more later)

Thoughts?